openstack/loadbalancer/v2/pools/requests.go

```diff
@@ -492,6 +493,35 @@ func BatchUpdateMembers(c *gophercloud.ServiceClient, poolID string, opts []Batc
 	return
 }
 
+// BatchUpdateMembers updates the pool members in batch with additive_only=true set
+// so there will be no deletion of existing members
+func BatchUpdateMembersAdditiveOnly(c *gophercloud.ServiceClient, poolID string, opts []BatchUpdateMemberOpts) (r UpdateMembersResult) {
+	members := []map[string]interface{}{}
+	for _, opt := range opts {
+		b, err := opt.ToBatchMemberUpdateMap()
+		if err != nil {
+			r.Err = err
+			return
+		}
+		members = append(members, b)
+	}
+
+	b := map[string]interface{}{"members": members}
```

Comment:
Looks like code duplication with `BatchUpdateMembers()` that you could minimize by extracting this into a separate function.